### PR TITLE
Add documentation to objects/nodebalancer

### PR DESCRIPTION
**Pull request by yec-akamai**
## 📝 Description

Add documentations to endpoint wrapper methods and resources classes under objects/nodebalancer.
https://jira.linode.com/browse/TPT-1930

### CODE REVIEW

File: linode_api4/objects/nodebalancer.py [hunk 1]
```diff
@@ -169,6 +212,16 @@ class NodeBalancer(Base):
 
     # create derived objects
     def config_create(self, label=None, **kwargs):
```
Comment (yec-akamai) [inline linode_api4/objects/nodebalancer.py:214]: It seems no `label` param in the current API doc https://www.linode.com/docs/api/nodebalancers/#config-create. Not sure if we should keep it or not.

Comment (ezilber-akamai) [reply linode_api4/objects/nodebalancer.py:214]: It looks like the `NodeBalancerConfig` class doesn't have a property called `label` either so I think it should be fine to remove it.

**Review by jriddle-linode — APPROVED** [review]

File: linode_api4/objects/nodebalancer.py (hunk 1 above)
Comment (zliang-akamai) [reply linode_api4/objects/nodebalancer.py:214]: I agree

File: linode_api4/objects/nodebalancer.py [hunk 2]
```diff
@@ -95,6 +115,23 @@ def nodes(self):
...
+        on their status.
+
+        API documentation: https://www.linode.com/docs/api/nodebalancers/#node-create
+
+        :param address: The private IP Address where this backend can be reached.
+                        This must be a private IP address.
+        :type address: str
+
+        :param label: The label for this node. This is for display purposes only.
+        :type label: str 3..32 characters
```
Comment (zliang-akamai) [inline linode_api4/objects/nodebalancer.py:130]:
```suggestion
        :param label: The label for this node. This is for display purposes only. Length: 3 to 32.
        :type label: str
```

**Review by zliang-akamai — APPROVED** [review]

File: linode_api4/objects/nodebalancer.py (hunk 1 above)
Comment (yec-akamai) [reply linode_api4/objects/nodebalancer.py]: Removed it.

**Review by ezilber-akamai — APPROVED** [review]

**Pull request merged**